app/scaffold-controllers/welcome-screen/welcomeModalController.js

```diff
@@ -0,0 +1,87 @@
+define([
+    'astro-full',
+    'astro-rpc',
+    'bluebird',
+    'plugins/modalViewPlugin',
+    'plugins/secureStorePlugin',
+    'scaffold-controllers/welcome-screen/welcomeController'
+],
+/* eslint-disable */
+function(
+    Astro,
+    AstroRpc,
+    Promise,
+    ModalViewPlugin,
+    SecureStorePlugin,
+    WelcomeController
+) {
+/* eslint-enable */
+
+    var WelcomeModalController = function(modalView) {
+        this.isShowing = false;
+        this.viewPlugin = modalView;
```

Comment:
So this is one place where I don't think we should use the `viewPlugin` convention. This is basically a modal view and we wouldn't integrate it into any view hierarchy. We should probably just call this `modalView`.